lms/static/sass/partials/base/_variables.scss

```diff
@@ -247,6 +247,7 @@ $m-blue-d3: rgb(18,111,154);
 $m-blue-d4: rgb(10,74,103);
 $m-blue-d5: rgb(0,158,231);
 $m-blue-d6: #256A97;
+$m-blue-d7: #0079BC;
```

Comment:
This shouldn't be necessary with the switch to using the non marketing site prefixed blue variables.